### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by dl239**
* **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +29,10 @@ FEDB is a NewSQL database optimised for realtime inference and decisioning appli
 
 Note: The latest released FEDB is unstable and not recommend to be used in production environment.
 
+### Architecture
```
Comment (tobegit3hub) [inline README.md:32]: 这一段还是太简单了，要不直接合并到introduction里。

File: README.md [hunk 2]
```diff
@@ -48,9 +54,24 @@ mkdir -p build && cd build && cmake ../ && make -j5 fedb
 * Detect the healthy of online transaction and make an alert -oncoming
 * Online real-time transaction fraud detection -oncoming
 
-## Architecture
+## Performance
+
+In AI scenarios most real-time features are time-related and required to be computed over multiple time windows. So we use computation TopN queries as benchmark scenary.
```
Comment (tobegit3hub) [inline README.md]: 最后的scenary应该改成scenario。

**Review by tobegit3hub — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -48,9 +54,24 @@ mkdir -p build && cd build && cmake ../ && make -j5 fedb
 * Detect the healthy of online transaction and make an alert -oncoming
 * Online real-time transaction fraud detection -oncoming
 
-## Architecture
+## Performance
```
Comment (tobegit3hub) [inline README.md:57]: 按今天讨论，Performance可以挪到QuickStart后面。

File: README.md (hunk 1 above)
Comment (dl239) [reply README.md:32]: 现在就是作为introduction的一个子项了，三个#号哈

File: README.md (hunk 2 above)
Comment (dl239) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (dl239) [reply README.md:57]: 嗯

**Pull request merged**